File: benchmark/stress/bench_call.c
#include <stdio.h>
#include <time.h>

static int inc(int x) { return x + 1; }

int main(void) {
    struct timespec s, e;
    clock_gettime(CLOCK_MONOTONIC, &s);
    int v = 0;
    for (int i = 0; i < 1000000; i++) v = inc(v);
    clock_gettime(CLOCK_MONOTONIC, &e);
    double ms = (e.tv_sec-s.tv_sec)*1000.0 + (e.tv_nsec-s.tv_nsec)/1e6;
    printf("result: %d  time: %.2fms\n", v, ms);
    return 0;
}

File: benchmark/stress/bench_call.rs
use std::time::Instant;

fn inc(x: i32) -> i32 { x + 1 }

fn main() {
    let start = Instant::now();
    let mut v = 0i32;
    for _ in 0..1_000_000 { v = inc(v); }
    let ms = start.elapsed().as_secs_f64() * 1000.0;
    println!("result: {}  time: {:.2}ms", v, ms);
}
